modules/manage/pages/manage-xdcr/xdcr-differ.adoc

```diff
@@ -0,0 +1,58 @@
+= xdcrDiffer Utility
+:description: pass:q[The `xdcrDiffer` utility helps you verify data consistency between XDCR clusters by comparing document metadata and values, and reporting missing or mismatched documents.]
+
+[abstract]
+{description}
+
+`xdcrDiffer` is a diagnostic utility included in the Server installation package for convenient access.
+Previously, you'd build this utility from source using the xdcrDiffer GitHub repository.
+
+== What is xdcrDiffer?
+
+`xdcrDiffer` is a Diff tool that helps you check data consistency between XDCR clusters.
+For full usage instructions, see the https://github.com/couchbase/xdcrDiffer[xdcrDiffer documentation].
```

Comment:
Just a small change to emphasize that there are Q&A's on the page as well.  I thought that it was better to take the link to the Readme section so that they'll see the info without having to scroll down or click on Readme on the side panel.  Also, thought it was better to just call it a Readme (which is what it is) so that people aren't confused.

For full usage instructions and Q&A's, see the https://github.com/couchbase/xdcrDiffer?tab=readme-ov-file#readme[xdcrDiffer Readme].